packages/web/src/ui/views/viewer.js

```diff
@@ -58,6 +58,8 @@ const viewer = (state, i18n) => {
     render = prepareRender(viewerOptions)
     const gestures = pointerGestures(el)
 
+    if(typeof window != 'undefined') window.addEventListener('resize', evt=>updateView = true)
```

Comment:
Please adjust to follow syntax rules for spacing, functions, etc.

Reply:
ok, forgot about that :) .. I have applied standardx to the file, also removed `if(typeof window != 'undefined')` as window global is used in few more places.